Library/Homebrew/language/haskell.rb

```diff
@@ -92,5 +92,50 @@ def install_cabal_package(*args, **options)
         end
       end
     end
+
+    module CabalV2
+      def cabal_sandbox(options = {})
+        pwd = Pathname.pwd
+        home = options[:home] || pwd
```

Comment:
When would/is `:home` manually set?

Reply:
I don't know.  I cargo-culted this from the v1-code, which I assumed was there for a reason.

Reply:
Ok, I'd say remove anything you don't specifically know is needed.